wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/rapidreactcommandbot/subsystems/Shooter.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.examples.rapidreactcommandbot.subsystems;
+
+import static edu.wpi.first.wpilibj2.command.Command.waitUntil;
+import static edu.wpi.first.wpilibj2.command.CommandGroupBase.parallel;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.examples.frisbeebot.Constants.ShooterConstants;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+  private final PWMSparkMax m_shooterMotor = new PWMSparkMax(ShooterConstants.kShooterMotorPort);
+  private final PWMSparkMax m_feederMotor = new PWMSparkMax(ShooterConstants.kFeederMotorPort);
+  private final Encoder m_shooterEncoder =
+      new Encoder(
+          ShooterConstants.kEncoderPorts[0],
+          ShooterConstants.kEncoderPorts[1],
+          ShooterConstants.kEncoderReversed);
+  private final SimpleMotorFeedforward m_shooterFeedforward =
+      new SimpleMotorFeedforward(
+          ShooterConstants.kSVolts, ShooterConstants.kVVoltSecondsPerRotation);
+  private final PIDController m_shooterFeedback = new PIDController(ShooterConstants.kP, 0, 0);
+
+  /** The shooter subsystem for the robot. */
+  public Shooter() {
+    m_shooterFeedback.setTolerance(ShooterConstants.kShooterToleranceRPS);
+    m_shooterEncoder.setDistancePerPulse(ShooterConstants.kEncoderDistancePerPulse);
+
+    // Set default command to turn off both the shooter and feeder motors, and then idle
+    setDefaultCommand(
+        runOnce(
+                () -> {
+                  m_shooterMotor.disable();
+                  m_feederMotor.disable();
+                })
+            .andThen(run(() -> {}))
```

Comment:
This seems sketchy, and is the only thing causing this config to not throw (unless we move forward with #4411). It seems very easy for a team to forget or remove this line, and end up with an exception.

Besides, are we sure that `run`, `runOnce`, etc will pick up on the inherited method from `Subsystem` and not the static factory from `Command` (via a star import -- specific imports seem less likely). This point in general makes me worry that teams might forget requirements by using the wrong method.